dask_ml/linear_model/utils.py

```diff
@@ -59,3 +59,13 @@ def add_intercept(X):  # noqa: F811
     if "intercept" in columns:
         raise ValueError("'intercept' column already in 'X'")
     return X.assign(intercept=1)[["intercept"] + list(columns)]
+
+
+@dispatch(np.ndarray)  # noqa: F811
```

Comment:
We can actually just use `np.vstack` now, since Dask implemented [NEP 18](https://numpy.org/neps/nep-0018-array-function-protocol.html).